cmd/cluster-config-operator/main.go

```diff
@@ -1,37 +1,21 @@
 package main
 
 import (
-	goflag "flag"
-	"fmt"
-	"math/rand"
 	"os"
-	"time"
 
 	"github.com/spf13/cobra"
-	"github.com/spf13/pflag"
 
-	utilflag "k8s.io/component-base/cli/flag"
-	"k8s.io/component-base/logs"
+	"k8s.io/component-base/cli"
 
 	"github.com/openshift/cluster-config-operator/pkg/cmd/operator"
 	"github.com/openshift/cluster-config-operator/pkg/cmd/render"
 	"github.com/openshift/cluster-config-operator/pkg/version"
 )
 
 func main() {
-	rand.Seed(time.Now().UTC().UnixNano())
-
-	pflag.CommandLine.SetNormalizeFunc(utilflag.WordSepNormalizeFunc)
-	pflag.CommandLine.AddGoFlagSet(goflag.CommandLine)
```

Comment:
I don't see component-base/cli adding the golang flag set to catch libraries that make use of them.

Reply:
That should be done during this init function https://github.com/kubernetes/component-base/blob/v0.23.0/logs/config.go#L49

Reply:
reverted back start command to simplify PR